projects/packages/blaze/src/class-dashboard-rest-controller.php

```diff
@@ -890,4 +899,21 @@ private function is_user_connected() {
 	private function get_site_id() {
 		return Connection_Manager::get_site_id();
 	}
+
+	/**
+	 * Check if the posts are synced.
+	 *
+	 * @return bool True if is sync, false otherwise.
+	 */
+	private function is_posts_synced(): bool {
+
+		/** The Full Sync Module.
+		 *
+		 * @var \Automattic\Jetpack\Sync\Modules\Full_Sync $full_sync_module
+		 */
+		$full_sync_module = Modules::get_module( 'full-sync' );
```

Comment:
Ensure the Full Sync Module is present and make phan happy :)
```suggestion
		$full_sync_module = Modules::get_module( 'full-sync' );
		'@phan-var Modules\Full_Sync_Immediately $full_sync_module';
		if ( ! $full_sync_module ) {
		    return true;
		}
```